apm-sniffer/apm-sdk-plugin/sharding-sphere-3.x-plugin/pom.xml

```diff
@@ -45,19 +45,19 @@
         <dependency>
             <groupId>io.shardingsphere</groupId>
             <artifactId>sharding-core</artifactId>
-            <version>[3.0.0,4.0.0)</version>
+            <version>[3.0.0,)</version>
```

Comment:
I think shardingsphere is going to change package name. I doubt this is the right move. Thoughts? @terrymanu

Reply:
Yes, you are right. 4.x should be apache release.